Raven.Database/Raft/Util/RaftHelper.cs

```diff
@@ -31,7 +31,7 @@ public static NodeConnectionInfo GetLeaderNode(this RaftEngine engine, int waitT
         {            
             if (waitTimeoutInSeconds > 0)
             {
-                if (engine.WaitForLeader(waitTimeoutInSeconds * 1000) == false)
+                if (engine.WaitForLeader(waitTimeoutInSeconds) == false)
```

Comment:
@maximburyak are you sure? https://github.com/ravendb/ravendb/blob/v3.5/Rachis/Rachis/RaftEngine.cs#L521

Reply:
The parameters name is waitTimeoutInSeconds, I'm sure that from the controllers point of view, I should pass seconds and not multyply by thousands. We'll have to make sure that no code path calls this method with miliseconds though

Reply:
`GetLeaderNode` parameter is called `waitTimeoutInSeconds` and yes, it expects the value to be in seconds, but `WaitForLeader` expects the value to be in milliseconds, so there is need of conversion here. Right?

Reply:
Yes, I think so. It's good  you  paid attention to that!